src/xcode/ENA/ENA/Source/HTTPClientCore/Service/Resource/_Resource.swift

```diff
@@ -32,11 +37,25 @@ protocol Resource {
 // Custom error handling & caching support
 
 extension Resource {
-	
+
+	// empty default model range
+	var defaultModelRange: [Int] {
+		[]
+	}
+
+	// empty default model
 	var defaultModel: Receive.ReceiveModel? {
 		nil
 	}
 
+	// if no default model range is give we always is the default model
+	func useFallBack(_ statusCode: Int) -> Bool {
+		if defaultModelRange.isEmpty {
+			return true
+		}
+		return defaultModelRange.contains(statusCode)
```

Comment:
Is it correct that the available hours call does not have a default model handling for all other errors that are != 404?

Reply:
I double check it, yes it is correct.